content/en/docs/components/pipelines/sdk/connect-api.md

```diff
@@ -204,13 +205,12 @@ Cross-namespace access can be achieved in two ways:
   Specifically, the owner of the `namespace-2` uses Kubeflow UI "Manage contributors" page. In the "Contributors to your namespace" 
   textbox he adds email address associated with the `namespace-1`.
 
-##### How Multi-User mode in-cluster authentication works
+#### How Multi-User mode in-cluster authentication works
 
-Authentication uses ServiceAccountToken 
-[projection](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection). 
-Simply put, the token is first being injected into a Pod (e.g. Jupyter notebook's). 
-Then Kubeflow Pipelines SDK uses this token to authorize against Kubeflow Pipelines API.
+When calling Kubeflow Pipelines API in the same cluster, Kubeflow Pipelines SDK authenticates itself as `default-editor` in your namespace using ServiceAccountToken 
```

Comment:
nit: instead of `default-editor`, it's more accurate to say "your Pod's service account" (or any other words with the same meaning), because the example is using `default-editor`, but people can configure auth with any other service accounts too.